assets/pages/digital-subscription-checkout/components/checkoutForm.jsx

```diff
@@ -76,7 +91,9 @@ function CheckoutForm(props: PropTypes) {
         type="tel"
         value={props.telephone}
         setValue={props.setTelephone}
+        error={firstError('telephone', props.errors)}
       />
+      <button onClick={() => props.submitForm()}>Submit</button>
```

Comment:
can we make this a regular `<form>` element using `onSubmit`? That way we get some a11y goodness plus you can the enter key after the last field